recipe/meta.yaml

```
@@ -108,41 +94,30 @@ outputs:
       build:
         - {{ compiler('cxx') }}
         - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
-        - autoconf                # [not win]
-        - automake                # [not win]
-        - libtool                 # [not win]
         - cmake
         - libgomp                 # [linux]
-        - llvm-openmp             # [osx or win]
-        - autotools_clang_conda   # [win]
+        - llvm-openmp             # [osx]
       host:
         - libblas
         - liblapack
-      run:
-        - libblas
-        - liblapack
       run_constrained:
         - faiss-proc * {{ faiss_proc_type }}
 
     test:
       commands:
         # shared
-        - test -f $PREFIX/lib/libfaiss.so                           # [linux]
-        - test -f $PREFIX/lib/libfaiss.dylib                        # [osx]
-        - if not exist %PREFIX%\\Library\\bin\\libfaiss.dll exit 1  # [win]
+        - test -f $PREFIX/lib/libfaiss.so                        # [linux]
+        - test -f $PREFIX/lib/libfaiss.dylib                     # [osx]
+        - if not exist %PREFIX%\\Library\\bin\\faiss.dll exit 1  # [win]
 
         # absence of static libraries
-        - test ! -f $PREFIX/lib/libfaiss.a                          # [not win]
-        - if exist %PREFIX%\\Library\\lib\\libfaiss.lib exit 1      # [win]
-
-        # conda tools
-        - conda inspect linkages -p $PREFIX $PKG_NAME               # [not win]
-        - conda inspect objects -p $PREFIX $PKG_NAME                # [osx]
+        - test ! -f $PREFIX/lib/libfaiss.a                       # [not win]
+        - if exist %PREFIX%\\Library\\lib\\faiss.lib exit 1      # [win]
 
         # headers
         {% for each_header in headers %}
-        - test -f $PREFIX/include/faiss/{{ each_header }}                                    # [unix]
-        - if not exist %LIBRARY_INC%\\faiss\\{{ "\\".join(each_header.split("/")) }} exit 1  # [win]
+        - test -f $PREFIX/include/faiss/{{ each_header }} || (echo "{{ each_header }} not found" && exit 1)  # [unix]
+        - if not exist %LIBRARY_INC%\\faiss\\{{ "\\".join(each_header.split("/")) }} exit 1                  # [win]
```

```suggestion

        - if not exist %PREFIX%\Library\bin\faiss.dll exit 1  # [win]

        # absence of static libraries
        - test ! -f $PREFIX/lib/libfaiss.a                       # [not win]
        - if exist %PREFIX%\Library\lib\faiss.lib exit 1      # [win]

        # headers
        {% for each_header in headers %}
        - test -f $PREFIX/include/faiss/{{ each_header }} || (echo "{{ each_header }} not found" && exit 1)  # [unix]
        - if not exist %LIBRARY_INC%\faiss\{{ "\\".join(each_header.split("/")) }} exit 1                  # [win]
```
^-- best to double check this; editing diffs in GitHub comments is.. let's say "non-optimal" XD

---

Erm... IDK what happened there. Diffing the wrong base or something..